core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3469,4 +3472,103 @@ class ReplicaManagerTest {
       ClientQuotasImage.EMPTY
     )
   }
+
+  def assertFetcherHasTopicId[T <: AbstractFetcherThread](manager: AbstractFetcherManager[T], tp: TopicPartition, expectedTopicId: Option[Uuid]): Unit = {
```

Comment:
nit: Could we split the arguments over multiple lines here? The line is quite long.